docs/reference/db-config.rst

```diff
@@ -12,17 +12,24 @@ Configuration properties
   :filter: Used
 
 
-Deprecated properties
----------------------
+Unused properties
+-----------------
 
 As ScyllaDB evolves, certain configuration properties may become obsolete or redundant.
 In our commitment to ensure smooth transitions, we have opted to keep this properties documented.
 Even though ScyllaDB does not use their values, we are maintaining them for anyone using older configurations that reference these options.
 
-Here is the list of deprecated properties:
+Here is the list of unused properties:
 
 .. dbconfigdatatemplate:: ../_data/db_config.yaml
   :template: db_config.tmpl
   :filter: Unused
 
+Invalid properties
```

Comment:
Wow,  I didn't even realize we had those "Invalid" options. Why do we even have them??? If we just dropped them, wouldn't the user also get an error when using them?

Reply:
Considering that ScyllaDB docs are versioned, would you prefer to remove the invalid properties from this document? And, the unused?

Reply:
Removed the complete section